BUILDING.txt

```diff
@@ -21,38 +21,15 @@ The easiest way to get an environment with all the appropriate tools is by means
 of the provided Docker config.
 This requires a recent version of docker (1.4.1 and higher are known to work).
 
-On Linux:
+On Linux or Mac:
     Install Docker and run this command:
 
     $ ./start-build-env.sh
 
-On Mac:
-    First make sure Virtualbox and docker toolbox are installed.
-    You can use docker toolbox as described in http://docs.docker.com/mac/step_one/.
-    $ docker-machine create --driver virtualbox \
-                            --virtualbox-memory "4096" hadoopdev
-    $ eval $(docker-machine env hadoopdev)
-    $ ./start-build-env.sh
-
-The prompt which is then presented is located at a mounted version of the source tree
-and all required tools for testing and building have been installed and configured.
-
 Note that from within this docker environment you ONLY have access to the Hadoop source
 tree from where you started. So if you need to run
     dev-support/bin/test-patch /path/to/my.patch
 then the patch must be placed inside the hadoop source tree.
-
-Known issues:
-- On Mac with Boot2Docker the performance on the mounted directory is currently extremely slow.
-  This is a known problem related to boot2docker on the Mac.
-  See:
-    https://github.com/boot2docker/boot2docker/issues/593
-  This issue has been resolved as a duplicate, and they point to a new feature for utilizing NFS mounts
-  as the proposed solution:
-    https://github.com/boot2docker/boot2docker/issues/64
-  An alternative solution to this problem is to install Linux native inside a virtual machine
-  and run your IDE and Docker etc inside that VM.
```

Comment:
You would need to restore these lines as Hadoop 2.10.x is now buildable using `Dockerfile` on Virtualbox (as of this commit).

Reply:
@GauthamBanasandra but when this PR merged, at some point, we can build hadoop 2.10.0 just with docker directly. And I don't know when the BUILDING.txt should change and how to describe that.

Reply:
I think we should retain the ability to build using Virtualbox and your PR would just add the ability to build using Docker, instead of deprecating the ability to build using Virtualbox. With that said, you wouldn't need to modify the docs here.

Reply:
ok, I will revert the file.